nixos/modules/programs/espanso-capdacoverride/default.nix

```diff
@@ -0,0 +1,60 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
+{
+  meta = {
+    maintainers = with maintainers; [ pitkling ];
+  };
+
+  options = {
+    programs.espanso.capdacoverride = {
+      enable = (mkEnableOption "espanso-wayland overlay with DAC_OVERRIDE capability") // {
+        default = false;
+        extraDescription = ''
+          Creates an espanso binary with the DAC_OVERRIDE capability (via `security.wrappers`) and overlays `pkgs.espanso-wayland` such that self-forks call the capability-enabled binary.
+          Required for `pkgs.espanso-wayland` to work correctly if not run with root privileges.
+        '';
+      };
+
+      package = mkOption {
+        type = types.package // {
+          check = package: types.package.check package && (builtins.elem "wayland" package.buildFeatures);
+          description =
+            types.package.description
+            + " for espanso with wayland support (`package.builtFeatures` must contain `\"wayland\"`)";
+        };
+        default = pkgs._espanso-wayland-orig;
+        defaultText = "pkgs.espanso-wayland (before applying the overlay)";
+        description = "The espanso-wayland package used as the base to generate the capability-enabled package.";
+      };
+    };
+  };
+
+  config =
+    let
+      cfg = config.programs.espanso.capdacoverride;
+    in
```

Comment:
This should be moved one scope higher like other modules

Reply:
done